docs/userguide/input.md

```diff
@@ -13,10 +13,13 @@ Every tracking experiment requires data, which you might need to download.
 
 * **Tracking region**
 
+```{eval-rst}
     In addition to the time-dependent forcing data, WAM2layers also needs a
-    netcdf-file containing a variable called `tagging_region`, with values between 0
-    and 1. It must have the same lat/lon coordinates as the other input data. This
-    file is used to define the tagging region from where to track moisture.
+    tagging region. This is used to define the tagging region from where to 
+    track moisture.
+    The region can be described by a netCDF file, shapefile, or simple
+    bounding box. For more info see :class:`wam2layers.config.Config.tagging_region`
+```
```

Comment:
Alternatively, you may find other options for building your own tagging region under notebooks.